content-src/components/Search/_Search.scss

```diff
@@ -51,7 +51,6 @@
     box-shadow: $shadow-secondary, 0 0 0 1px $black-25;
   }
 
-  &:active input,
```

Comment:
This was added in #3588 along with a related style:
https://github.com/mozilla/activity-stream/blob/6a2252ccf8e0aea02857db1fb7848802ead4c491/content-src/components/Search/_Search.scss#L84-L85

This is so that when the user is clicking on the search button, it adds the blue border/shadow and the depressed button.

Reply:
I guess i’ll use the new div.search-inner-wrapper

Reply:
Fixed. Now it seems to behave just like release. @Mardak